library.properties

```diff
@@ -5,5 +5,5 @@ maintainer=Microsoft <iotcert@microsoft.com>
 sentence=Azure C shared utility library for Arduino. For the Arduino MKR1000 or Zero and WiFi Shield 101, Adafruit Huzzah and Feather M0, or SparkFun Thing.
 paragraph=Microsoft port of the Azure C Shared Utility. Together with AzureIoTHub, it allows you to use your Arduino with the Azure IoT Hub.  See readme.md for more details.  Copyright (c) Microsoft. All rights reserved.  Licensed under the MIT license. See LICENSE file in the project root for full license information.
 category=Communication
-url=https://github.com/Azure/azure-iot-arduino-utility
-architectures=samd,esp8266
+url=https://github.com/zzeekk/azure-iot-arduino-utility
```

Comment:
@zzeekk - Can you please address this comment and submit update to the PR? Thanks.